client/src/components/Dashboard/Dashboard.css

```diff
@@ -1,15 +1,81 @@
-.dashboard .playlist-container {
+.dashboard {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: var(--background-color);
+  padding: 0 5%;
+  min-height: 100vh;
+}
+
+.dashboard .dashboard-right {
+  padding-left: 500px;
+  align-self: flex-start;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.dashboard .playlist-tab {
+  width: 100%;
+  padding: 10px 10px;
+  border-radius: 20px;
+  display: none;
+}
+
+.dashboard .favorites-tab {
+  width: 100%;
+  padding: 10px 10px;
+  display: none;
+}
+
+.dashboard .carousel {
+  width: 100%;
+  background: url('/Users/kimlinh/CodePath/Project/Capstone-Project/client/img/lily.jpeg');
+  background-size: cover;
+  background-position: 5rem;
+  height: 35vh;
```

Comment:
Because vh is based on the viewport size but % is based on the size of the parent, so I think I just used vh here b/c I wanted to scale it off of the size of the viewport.

Reply:
Gotcha. That makes sense.